requirements.master.txt

```diff
@@ -1 +1,3 @@
 tx-manager>=0.2,<0.3
```

Comment:
yeah, I was thinking that production should be pulled from github, but then would we need to always specify a particular tag number?

Also, do we need to do anything in setup.py for GitHub dependencies @phillip-hopper, or does just running `pip install -r requirements.txt` before runing setup.py enough? I never understand what looks at setup.py to install dependencies.